### DISCUSSION TITLE
Extra CountBadge for incoming changes

### DISCUSSION THREAD

**Discussion by iishika-s** [microsoft/vscode-discussions#720] [Extension Development QnA] [answered]
Hi, I want to show count of incoming changes in SCM view just like its done for outgoing changes. 
Is it possible to get another count badge for this?

![Screenshot 2023-07-27 142035](https://github.com/microsoft/vscode-discussions/assets/93257779/583a93d3-6d9c-4e9f-8ef4-418a6f0ba721)

like the outgoing count in the  image I want to have another badge for incoming

**Comment by isidorn** [comment, marked as answer]
This is currently not possible to be contributed by an extension. Thanks
